web/templates/note/form.html.eex

```diff
@@ -18,5 +18,3 @@
 
   <%= submit @submit_text %>
 <% end %>
-
-<%= link "Cancel", to: note_path(@conn, :index)%>
```

Comment:
Why not keep the html link in this template, but pass in a `cancel_link`? That would reduce some duplication (the fact that a form has a link, the link text, any classes that may change) so that if this ever changes you can be sure that it will remain consistent across the edit and new pages.